### ISSUE TITLE
Create "team" page

### ISSUE THREAD

**Issue by dcts** [DeXter-on-Radix/website#457]
Currently there is not much information available about the contributors who are actively working on dexter, which from an investors perspective, might be off putting. 

I think it would be great to have a dedicated page, where we show all active (and past) team members (or contributers), with the option to filter or search. We could leverage the [database about voting results](https://github.com/DeXter-on-Radix/voting-app/blob/main/dbibpvelmuqrvu.sql) to create badges (depending on tokens received) and add an "active" badge to all contributors that received votes within the last 3 phases.

Some additional ideas:
- show how many contributors there are in total
- contributors could update their profile (add text and links) and self declare their area of work (e.g. marketing, development, design, community work etc...). 
- show charts with additional insights (active contributors over time, distribution of work split among areas of work)

**Comment by yuliaSharabi** [comment]
Is this not done yet IYO? @dcts 
I think update profiles and charts should be a seperate Issues, not part of this one. 
The initial Team page is Live as far as I know.

**Comment by dcts** [comment]
Yes its done, closing! 

The updating profiles is a feature that will be possible only after the voting app redesign and is part of voting app specs, so no need for tracking separately.
